### DISCUSSION TITLE
Nur Domainname "fritz.box", nicht mehr...

### DISCUSSION THREAD

**Discussion by schabau** [Freetz-NG/freetz-ng#322] [Allgemein]
Zunächst einmal Hallo an alle,

nach ein paar Tagen basteln habe ich Freetz auf meine Fritzbox 7490 (originales AVM Model) bekommen.
Meine Intention war nicht anderes, als den von der Fritzbox fest vergebenen Domainname "fritz.box" zu ändern. Alles was die Fritzbox in Betreff DNS und DHCP ohnehin macht, reicht für mein kleines Netzwerk vollends aus.
Ich hatte mir "dnsmasq" kurz angesehen, möchte es aber eigentlich gar nicht nutzen, weil es viel zu viel ist.

Weiß jemand einen einfachen Weg, nur den von der Fritzbox vergebenen Domainnamen "fritz.box" zu ändern?
Ich habe in der Hinsicht etwas über die /var/flash/ar7.cfg Datei gelesen, doch die finde ich hier gar nicht.

Viele Grüße
Detlef Paschke

**Comment by fda77** [comment]
Einstellungs-export+-import

**Reply by schabau** [reply]
> Einstellungs-export+-import

Hallo,

zunächst vielen Dank für die Hilfe.

Ich weiß, Fragen werden in allen Freetz Foren oft als "zu wenig oder unzureichend bemüht" interpretiert, ich frage trotzdem noch einmal nach. Ich habe sehr lange Forenbeiträge gelesen allein um Freetz auf eine "jungfreudige" Fritzbox zu bekommen.

Ich hatte eine komplette Sicherung über die Freetz GUI angefertigt. In dieser *.tar befindet sich unter anderem eine settings.tgz und wenn ich tiefer steige finde ich in /var_flash/ Tatsächlich die genannte ar7.cfg.

Wenn ich in dieser nun unter "dhcpc_hostname" die gewünschte Domain eintrage und die Sicherung zurückschreibe, bleibt sie auch über einen Neustart hinaus bestehen?

Viele Grüße
Detlef Paschke

**Comment by fda77** [comment]
Achso, mit Freetz. Du kannst die Datei mit "nvi" auf derm Gerät bearbeiten. 
Wenn es nach einem Reboot verschwunden ist, hat der ctlmgr sein Cache zurückgeschrieben
https://github.com/Freetz-NG/freetz-ng/blob/master/docs/make/mod.md#udevmount

**Reply by schabau** [reply]
> Achso, mit Freetz.

Schrieb ich das nicht und geht es hier nicht um Freetz. Habe ich mich wohl falsch ausgedrückt. Sorry.
Den Link schaue ich mir Morgen mal an.

**Comment by joerg-wille** [comment]
Duplikat #924 
@schabau Hast Du einen Weg gefunden, nur den dhcpc_hostname zu verändern und ist die Änderung auch persistent?
Den Tipp von @fda77 kann ich leider nicht ganz nachvollziehen. 
Meine Annahme:
Die Variable dhcpc_hostname steht wohl in ar7.cfg und die kann man mittels des Skripts "nvi", bzw. mittels des Skripts "wrap_editors" - nvi ist (mittlerweile) nur noch ein link auf letzteres Skript - anpassen.
Da diese Änderung aber nicht persistent ist, soll man das beim Booten der Box Automatisieren - so jedenfalls verstehe ich den Hinweis auf udevmount.

Oder wie sonst kann man erreichen - am besten ohne auf dnsmasq zu wechslen?

**Comment by eMPee584** [comment]
Hatte das gleiche Anliegen, hab mein Netzwerk gerad von einem `OpenWRT`-Router zu einer 7490 `Freetz-NG`-box umgezogen und wollte dass die Geräte (Drucker u so) wieder unter den vorherigen Hostnamen erreichbar sind. Hatte gehofft, in `/etc/resolvconf/resolv.conf.d/` irgendwo die Zeile `search lan` einzufügen um das lokal zu regeln aber hat scheinbar nicht funktioniert.. : /
